src/content/web/wasm.md

```diff
@@ -41,6 +41,18 @@ restrictions:
    contain a temporary workaround. For details, see
    [webdev issue 2206]({{site.repo.dart.org}}/webdev/issues/2296).
 
+### Supported packages
+
+You can find packages that are compatible with Wasm by using the [`wasm-ready`][]
+filter on [pub.dev][].
```

Comment:
**issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
To find Wasm-compatible packages,
use the [`wasm-ready`][] filter on [pub.dev][].
```